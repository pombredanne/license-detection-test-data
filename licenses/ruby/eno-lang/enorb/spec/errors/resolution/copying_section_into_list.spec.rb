input = <<~DOC.strip
# original

# other

copy < original
- item
DOC

describe Eno::Errors::Resolution do
  describe '::copying_section_into_list' do
    let(:error) do
      intercept_parse_error { Eno.parse(input) }
    end

    it 'provides the correct message' do
      expect(error.message).to match_snapshot
    end

    it 'provides correct selection metadata' do
      expect(error.selection).to match_snapshot
    end
  end
end
