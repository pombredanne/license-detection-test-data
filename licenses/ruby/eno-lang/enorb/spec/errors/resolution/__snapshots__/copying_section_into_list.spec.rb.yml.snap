---
"./spec/errors/resolution/copying_section_into_list.spec.rb[1:1:1]": "In line 5 a
  section is copied into a list.\n\n   Line | Content\n   ...\n      3 | # other\n
  \     4 | \n >    5 | copy < original\n      6 | - item\n"
"./spec/errors/resolution/copying_section_into_list.spec.rb[1:1:2]":
- - 4
  - 0
- - 4
  - 15
